public/css/presenter.css

```diff
@@ -282,6 +282,15 @@
       #preview .thumb {
         display: none;
         position: absolute;
+        top: 0;
+      }
+        #preview .thumb .label {
```

Comment:
That's what the PR description insinuates :) This is where label styling will go when someone does it.